website/content/en/highlights/2022-03-31-native-event-codecs.md

```diff
@@ -26,6 +28,8 @@ command = ["./scrape.sh"]
 decoding.codec = "native_json"
 ```
 
+Running this configuration like:
```

Comment:
The "like" feels wrong here, since what follows isn't how to run the config. Maybe we move "Running the following config" above the config, then add "calling the following example script" above the bash. That seems to flow alright into "will send counters into Vector".